Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/Constants.java

```diff
@@ -5,17 +5,36 @@ public class Constants {
 
     public static final double CORRECTION_CONSTANT = 1 / 45.0; // constant for converting angle error to motor speed
 
-    public static final int SLIDE_HIGH = 2750;
-    public static final int SLIDE_MEDIUM = 1950;
-    public static final int SLIDE_LOW = 1150;
+    public static final int SLIDE_TOP = 3900;
+    public static final int SLIDE_HIGH = 3800;
+    public static final int SLIDE_MEDIUM = 2750;
+    public static final int SLIDE_LOW = 1650;
+    public static final int SLIDE_STOW = 600;
+    public static final int SLIDE_STACK_FOUR = 320;
+    public static final int SLIDE_STACK_THREE = 240;
+    public static final int SLIDE_STACK_TWO = 160;
+    public static final int SLIDE_STACK_ONE = 80;
```

Comment:
What are these stack numbers?

Reply:
the number of ticks the slide motors move up to reach the signified cone height for the autonomous stack